src/utils.ts

```diff
@@ -227,11 +234,11 @@ export async function getScores(
     };
     const res = await fetch(scoreApiUrl, {
       method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
+      headers: SCORE_API_HEADERS,
       body: JSON.stringify({ params })
     });
     const obj = await res.json();
-    return obj.result.scores;
+    return options.returnValue ? obj.result[options.returnValue] : obj.result;
```

Comment:
This is to remove this duplicate code https://github.com/snapshot-labs/snapshot-sequencer/blob/master/src/scores.ts#L46-L74 refer to the comment on top of the function